index4_4.js

```diff
@@ -0,0 +1,22 @@
+/*4.	Функция принимает объект с неограниченным количеством полей. Результатом выполнения функции должен быть объект, в котором были отсеяны поля со значениями null или undefined
+
+fn({ a: 1, b: ‘Hello’, c: null, z: undefined}) // { a: 1, b: ‘Hello’ }
+fn({ name: ‘alex’, age: 10, friends: [], address: null}) // { name: ‘alex’, age: 10, friends: [], }
+*/
+
+const filterNull = (data) => {
+  const arrayOfData = Object.entries(data);
+  const arrayWithoutNull = arrayOfData.filter((value) => {
+    return !(value[1] === null);
```

Comment:
1. use disctructuring instead of accessing by index
2. return typeof value !== 'undefined' && value !== null looks better